packages/jwt-middleware/index.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Assert = require('assert');
+const ExpressJwt = require('express-jwt');
+const jws = require('jws');
+const JwksRsa = require('jwks-rsa');
+
+const WT_AUTH_EXEC = 'wt-authorize-execution';
+
+module.exports = () => {
+
+    return function(req, res, next) {
+
+        const ctx = req.webtaskContext;
+    
+        const iss = ctx.meta['iss'];
+        const aud = ctx.meta['aud'];
```

Comment:
The middleware compiler's main function is taking an `options` object that contains the `meta`, so this can be captured once per middleware as opposed to per request.

Reply:
@ggoodman can you weigh in on this?  I don't think `options` is being passed to my middleware, therefore, I decided to use `module.webtask.meta` which does not depend on `req`.  You mentioned that `module.webtask` has issues and I should be using `req.webtaskContext` instead which is what I did above.

Reply:
Since `options` is not available in the middleware, the code is now using a factory function that will access `res.webtaskContext.meta` only on the initial call.